TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/traffic/expiration/ExpiringTrafficStreamMap.java

```diff
@@ -109,18 +109,19 @@ private boolean updateExpirationTrackers(String partitionId, String connectionId
         if (targetBucketHashSet == null) {
             var startOfWindow = expiringQueue.firstKey().toInstant();
             assert !timestampMillis.test(startOfWindow, (ts, windowStart) -> ts < windowStart) :
-                    "Only expected to not find the target bucket when the incoming timestamp was before the " +
+                    "Only expected to NOT find the target bucket when the incoming timestamp was before the " +
```

Comment:
Minor, but the double-ish negative is a bit confusing here. Is this equivalent to saying "Did not expect to find the target bucket when..."? Is the "only" adding additional information?

Reply:
Changed this to "Only expected the target bucket to be missing when ..."